Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -16,6 +16,7 @@
 #include "InputCommon/ControllerInterface/Xlib/XInput2.h"
 #endif
 #ifdef CIFACE_USE_OSX
+#include "InputCommon/ControllerInterface/GameController/GameController.h"
```

Comment:
Is this PR WIP? Init/PopulateDevices/DeInit aren't plugged in in the ControllerInterface.
I don't understand how you even tested this.

Reply:
I'm sorry for the confusion. I own an M1 Mac and had to test/develop in skylersaleh's fork as I am unable to build Dolphin on the main branch.
I forgot to manually copy the other changes I made in ControllerInterface.cpp